src/gridstack.ts

```diff
@@ -2111,18 +2111,14 @@ export class GridStack {
         if (!node) {
           node = this._readAttr(el, false); // don't wipe external (e.g. drag toolbar) attr #2354
         }
-        if (!node.grid) {
-          node._isExternal = true;
-          el.gridstackNode = node;
-        }
 
         // calculate the grid size based on element outer size
         helper = helper || el;
         let w = node.w || Math.round(helper.offsetWidth / cellWidth) || 1;
         let h = node.h || Math.round(helper.offsetHeight / cellHeight) || 1;
 
         // if the item came from another grid, make a copy and save the original info in case we go back there
-        if (node.grid && node.grid !== this) {
+        if (node.grid !== this) {

```

Comment:
still doesn't feel right to me without debuging the issue - if node doesn't have grid, we go through that logic which doesn't feel right...

Reply:
I agree with you. Now it seems to me that the problem is not in this place at all, but in calling the mouseEnter event in mouseOver. If you comment on this line as a definition of the source of the problem, then everything works correctly - https://github.com/gridstack/gridstack.js/blob/master/src/dd-droppable.ts#L135